### DISCUSSION TITLE
Stuck in queue

### DISCUSSION THREAD

**Discussion by benjamin-mogensen** [community/community#58508] [Actions] [answered]
We are seeing this issue that our actions get stuck in queue.

Runners normally queue for a few seconds, now don't get to run at all, just stuck in queue.

Using Ubuntu-latest.

Anyone else having issues??

Just to add to this - GitHub May be facing some issues with actions as I received email that the issue and project labeler action for this issue has failed to run.

**Comment by Akash1134** [comment, marked as answer]
Hi @benjamin-mogensen ,

Are you still facing the issue? It seems Actions were stuck due to an incident on July 19 - [details are here](https://www.githubstatus.com/incidents/22qm4j1kvn0h)

**Reply by benjamin-mogensen** [reply]
No, it went away after some hours. Let's close this one. Apologies for not writing back.
